### PR TITLE
Feature/tvos

### PR DESCRIPTION
**Pull request by dcharbonnier**
- add a tvOS target
- cleanup the tests setup (probably an old version of xcode)
- fix deprecation of NSURLConnection (are the tests covering that ?)
- cleanup on the framework path (no more warn on build)

### CODE REVIEW

File: Raven/RavenClient.swift [hunk 1]
```diff
@@ -400,7 +383,7 @@ public class RavenClient : NSObject, NSURLConnectionDelegate, NSURLConnectionDat
             }
         }
 
-        #if os(iOS)
+        #if os(iOS) || os(tvOS)
             if (tags["OS version"] == nil) {
```
Comment (dcharbonnier) [inline Raven/RavenClient.swift:387]: should we add `UIDevice.currentDevice().systemName` to make a difference between tvOS and iOS ? or
`tags["OS version"] = "\(UIDevice.currentDevice().systemName).\(UIDevice.currentDevice().systemVersion)"`

**Comment by dcharbonnier** [comment]
The test coverage is not very good, the request was not sent after my changes and the tests succeeded.
I've fixed the problem and tested manually with the example project, I received the message on my sentry.

**Comment by timorzadir** [comment]
Sweet. Thanks :)

File: Raven/RavenClient.swift (hunk 1 above)
Comment (timorzadir) [inline Raven/RavenClient.swift:387]: The `Device model` property would take care of the difference, wouldn't it?

Comment (dcharbonnier) [inline Raven/RavenClient.swift:387]: it's a different os but not a big deal, most of the people probably log on a different sentry project ios and apple tv

**Comment by dcharbonnier** [comment]
:+1:

**Pull request merged**